chains/chain-bridge-hub-polkadot/src/lib.rs

```diff
@@ -25,7 +25,7 @@ use bp_messages::*;
 use bp_runtime::{
 	decl_bridge_finality_runtime_apis, decl_bridge_messages_runtime_apis, Chain, ChainId, Parachain,
 };
-use frame_support::dispatch::DispatchClass;
+use frame_support::{dispatch::DispatchClass, weights::Weight};
```

Comment:
I think the `Weight` is coming from `bp_messages::*` there is a reexport - not sure if we want to keep this reexport or better remove it:
```
// Weight is reexported to avoid additional frame-support dependencies in related crates.
pub use frame_support::weights::Weight;
```

```suggestion
use frame_support::dispatch::DispatchClass;
```